src/crypto/convert.js

```diff
@@ -15,7 +15,7 @@
 import bignum from 'browserify-bignum';
 import ed2curve from 'ed2curve';
 import { getSha256Hash } from './hash';
-import getTransactionBytes from '../transactions/transactionBytes';
+import getTransactionBytes from '../transactions/utils/transactionBytes';
```

Comment:
How about exporting `getTransactionBytes` from an `index.js` in the `utils` dir and then doing this:
```js
import { getTransactionBytes } from '../transactions/utils';

```